ngraph/core/src/type/element_type.cpp

```diff
@@ -12,9 +12,9 @@
 #include "ngraph/log.hpp"
 #include "ngraph/type/element_type_traits.hpp"
 
-using namespace ngraph;
+using namespace ov;
```

Comment:
In general LGTM but I can see some fixes are needed to build. 
(example error: reference to ‘element’ is ambiguous)

Reply:
Yes of course I will have a look and fix issues 😄